docs/integration/squirrel.txt

```diff
@@ -64,4 +63,5 @@ and scripts.
 
 .. _SQuirreL: http://www.squirrelsql.org
 .. _Crate JDBC: https://crate.io/docs/projects/crate-jdbc/stable/
-.. _Crate JDBC standalone jar: https://bintray.com/crate/crate/crate-jdbc/view/files/io/crate/crate-jdbc-standalone
\ No newline at end of file
+.. _Crate JDBC standalone jar:
```

Comment:
just checked it, doesn't render correctly: 
![screenshot](https://cloud.githubusercontent.com/assets/38700/8569778/3041ca32-257b-11e5-91ff-d03e6f5c3df9.png)